business_objects/user.py

```diff
@@ -157,6 +157,7 @@ def get_active_users_after_filter(
     FROM public.user u 
     LEFT JOIN organization o
         ON u.organization_id = o.id
+    WHERE u.email IS NOT NULL
```

Comment:
with the condition below this will create a falty SQL statement (last_interaction needs an AND)